dlrover/python/master/elastic_training/net_topology.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 The DLRover Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABCMeta, abstractmethod
+from collections import OrderedDict
+from dataclasses import dataclass
+from typing import Dict, List, Tuple
+
+
+@dataclass
+class NodeTopologyMeta(object):
+    node_rank: int = 0
+    process_num: int = 0
+    node_ip: str = ""
+    asw: str = ""
+    psw: str = ""
+
+
+class TopologyQuerier(metaclass=ABCMeta):
+    @abstractmethod
+    def query(self, node_ip) -> Tuple[str, str]:
```

Comment:
According to the above, the ```query``` method definitions seems to be too specific in the network topology scenario.  Similar problems for other method definition.

Reply:
Also, for current method, should be named ```queryAswAndPsw```.